scripts/jest/preprocessor.js

```diff
@@ -18,7 +18,7 @@ const createCacheKeyFunction = require('fbjs-scripts/jest/createCacheKeyFunction
 const getBabelOptions = require('../getBabelOptions');
 const path = require('path');
 
-const {testSchemaPath} = require('../../dist/relay-test-utils');
+const testSchemaPath = require('../../dist/relay-test-utils/lib/RelayTestSchemaPath');
```

Comment:
This change fixes this test failure:

```
FAIL packages/react-relay/classic/mutation/__tests__/RelayMutationQuery-test.js
  ● Test suite failed to run
    Cannot find module 'relay-compiler'
      14 |     parse = _require.parse;
      15 | 
    > 16 | var _require2 = require("relay-compiler"),
         |                 ^
      17 |     Parser = _require2.Parser,
      18 |     convertASTDocuments = _require2.convertASTDocuments;
      19 | 
      at Object.<anonymous> (dist/relay-test-utils/lib/parseGraphQLText.js:16:17)
```

`relay-test-utils` imports some files from `relay-compiler` but it will not be able to resolve it from `dist`. This was working before, because the `require`'s were not actually run since they were inlined inside of function calls.

Reply:
👍